src/javascript/app_2/Stores/Modules/Contract/Helpers/__tests__/details.js

```diff
@@ -0,0 +1,99 @@
+import { expect }                           from 'chai';
+import React                                from 'react';
+import { getDetailsInfo, getDetailsExpiry } from '../details';
+import Money                                from "../../../../../App/Components/Elements/money";
+
+describe('Details', () => {
+    describe('getDetailsInfo', () => {
+        const contract_info = {
+            "buy_price":"10.00",
+            "contract_id":"36813286888",
+            "contract_type":"CALL",
+            "currency":"USD",
+            "current_spot":"8121.081",
+            "current_spot_time":1544029136,
+            "date_expiry":1544029145,
+            "date_settlement":1544029145,
+            "date_start":1544029135,
+            "entry_spot":"8121.081",
+            "is_expired":0,
+            "payout":"19.42",
+            "purchase_time":1544029135,
+        };
+        it('Should return an object with values based on object passed to it', () => {
+            expect(getDetailsInfo(contract_info)).to.deep.include({
+                'Contract Type' : 'Higher',
+                'Start Time'    : '2018-12-05 16:58:55 GMT',
+                'Entry Spot'    : '8,121.081',
+                'Purchase Price': <Money amount='10.00' currency='USD' />,
+            });
+        });
+    });
+
+    describe('getDetailsExpiry', () => {
+        it('Should return an empty object if store is passed to it and property "is_ended" is false', () => {
+            expect(getDetailsExpiry({is_ended: false})).to.deep.include({});
```

Comment:
If the desired return value SHOULD BE an empty object, we may use `to.be.equal` instead of non-strict `include`, Or better yet, we can check directly for empty objects using `expect({}).to.be.empty;`.